docs(guide): emphasize the install command is for setting a default version (#107)
volta-cli/website: subdomains/docs/_guide/understanding.md

You control the tools managed by your Volta toolchain with two commands: `volta install` and `volta uninstall`.

### Installing Node engines

> Every time you install a tool to your toolchain, you set the default version of that tool, which Volta will use unless you're working within a project directory that has configured Volta to use a different version.

Reviewer: I like this improvement. I wonder if we could make the different way of thinking about tools even more explicit, with something like:

```suggestion
To install a tool to your toolchain, you set the _default version_ of that tool, which Volta will use unless you're working within a project directory that has configured Volta to use a different version. When you choose a default version, Volta will also download that version to the local cache.
```
That inverts the emphasis so that instead of "downloading with a side effect of setting the default" it becomes "setting a default with a side effect of downloading", which is more accurate to how Volta acts. It also probably will fit better with a potential rename to `volta use`.

> For example, you can select your default version of `node` by installing a particular version:

Reviewer: With the above suggestion, could we also cut out the reference to "installing" here? Perhaps something like:

```suggestion
For example, you can select an exact version of `node` to be your default version:
```

Author: Yes I like these changes! In most apps/tools "when you install" usually means the first time,  so emphasizing that the command is primarily meant to set the default version clears things up. I incorporated your suggestions with a minor change of splitting a long winded sentence. Let me know if you have any additional feedback. And thanks for making Volta, it's great!
